fmriprep/workflows/bold/base.py

```diff
@@ -514,6 +514,37 @@ def init_bold_wf(
             (bold_anat_wf, bold_surf_wf, [('outputnode.bold_file', 'inputnode.bold_t1w')]),
         ])  # fmt:skip
 
+        if nonstd_spaces.intersection(('anat', 'T1w')):
+            # Source file should be output T1w-space volumetric file and fsnative2t1w_xfm
```

Comment:
I disagree. I see the use of this output as an implementation detail, and one I hope to move away from. We should be able to resample directly from BOLD to surface eventually.

Reply:
Fair enough. I'll drop that part.